### DISCUSSION TITLE
MTD cluster identification issue

### DISCUSSION THREAD

**Discussion by hertneky** [dtcenter/METplus#1391] [Configuration] [answered]
I setup and tested a MTD feature-relative use-case for a duration of 6 hours. It appeared to work fine. MTD identified the intended object cluster, for which the lat/lon attributes could then be used for tracking the feature. I then increased the run to extend over the full period of 48 hours. MTD no longer identified the object cluster. I literally changed nothing else, only:

LEAD_SEQ = begin_end_incr(1,6,1)  vs. LEAD_SEQ = begin_end_incr(1,48,1)

I am having trouble wrapping my head around why clusters are not identified for the longer run. The objects identified in the first 6 hours in both fcst/obs are basically identical for the 6- and 48-hour runs, but again no clusters for the 48-hour run. The MTD feature-relative uses the clusters for the tracks, so it is causing a problem. Any ideas on what might be causing this?

My runs can be found on seneca:
6hr run: /d1/personal/hertneky/metprd_06
48hr run: /d1/personal/hertneky/metprd_48
parm: /d1/personal/hertneky/parm/jtti_wpc_mtd_featureRelative.conf

**Comment by georgemccabe** [comment, marked as answer]
Hi @hertneky,

Your 6 hour run produced a pair with interest of 0.844 which is greater than the interest threshold of 0.7. Your 48 hour run produced no pairs with interest > 0.7.

On seneca:

 **cat /d1/personal/hertneky/metprd*/mtd/2020121612/mtd_20201216_130000V_3d_pair_simple.txt | awk '{print $35}'**

> INTEREST
> 0.844
> INTEREST
> 0.527
> 0.238
> 0.228
> 0.419
> 0.164
> 0.311
> 0.229
> 0.376
> 0.317
> 0.480

Since no pairs have interest > 0.7, there are no matched clusters.
The total interest is a weighted average of the interest values for each of the pair attributes. Several of those are based on the full length of the time series... so changing the time series length impacts the result. You could consider modifying the weights to make the INTEREST much less dependent on it... or just keep everything as is and change the total interest threshold to >0.5.

**Reply by hertneky** [reply]
Perfect explanation George. Thank you! I combed through the log file for answers, but had not thought of looking at the interest! You're absolutely right!
